java/vector/src/main/java/org/apache/arrow/vector/BitVectorHelper.java

```diff
@@ -18,6 +18,9 @@
 package org.apache.arrow.vector;
 
 import static org.apache.arrow.memory.util.LargeMemoryUtil.checkedCastToInt;
+import static siren.io.netty.util.internal.PlatformDependent.getByte;
+import static siren.io.netty.util.internal.PlatformDependent.getInt;
+import static siren.io.netty.util.internal.PlatformDependent.getLong;
```

Comment:
These can be removed, `PlatformDependent` is not used in arrow vector anymore, see https://github.com/apache/arrow/pull/14913

Reply:
Yes, thats correct. This has been fixed: 77b48e98d6c1c6da85d9cc9bcbd11a1f2f2eda63